### PR TITLE
Use frame id from received gps topic.

### PR DESCRIPTION
**Pull request by RonaldEnsing**
Related to #35.

### CODE REVIEW

**Comment by beetleskin** [comment]
Now we have two pull-requests fighting for the same issue, #35 :)

File: src/aerialmap_display.cpp [hunk 1]
```diff
@@ -114,7 +110,6 @@ AerialMapDisplay::~AerialMapDisplay()
 
 void AerialMapDisplay::onInitialize()
 {
-  frame_property_->setFrameManager(context_->getFrameManager());
```
Comment (beetleskin) [inline src/aerialmap_display.cpp]: why removing this?

File: src/aerialmap_display.cpp [hunk 2]
```diff
@@ -166,12 +161,6 @@ void AerialMapDisplay::updateAlpha()
...
 
-void AerialMapDisplay::updateFrame()
-{
-  ROS_INFO_STREAM("Changing robot frame to " << frame_property_->getFrameStd());
-  transformAerialMap();
-}
-
 void AerialMapDisplay::updateDrawUnder()
 {
   // @todo figure out why this property only applies to some objects
```
Comment (beetleskin) [inline src/aerialmap_display.cpp]: @schra any idea what this comment meant?

File: src/aerialmap_display.cpp [hunk 3]
```diff
@@ -494,7 +483,7 @@ void AerialMapDisplay::transformAerialMap()
   static std::string const mapFrame = "map";
```
Comment (beetleskin) [inline src/aerialmap_display.cpp]: could you add a comment that mentions why we have this constant? -> [REP 105 ](https://www.ros.org/reps/rep-0105.html)

File: src/aerialmap_display.cpp (hunk 1 above)
Comment (beetleskin) [reply src/aerialmap_display.cpp]: @schra what is it doing anyways?

File: src/aerialmap_display.cpp [hunk 4]
```diff
@@ -114,7 +110,6 @@ AerialMapDisplay::~AerialMapDisplay()
 
 void AerialMapDisplay::onInitialize()
```
Comment (beetleskin) [inline src/aerialmap_display.cpp]: you can remove this function altogether then

**Comment by beetleskin** [comment]
could you add a changelog entry, and run clang-format?

**Comment by RonaldEnsing** [comment]
Checked 'allow edits from maintainers'.

File: src/aerialmap_display.cpp (hunk 3 above)
Comment (schra) [reply src/aerialmap_display.cpp]: Let's do this in the PR for #54 and not in this PR

File: src/aerialmap_display.cpp (hunk 2 above)
Comment (schra) [reply src/aerialmap_display.cpp]: I do not know what Gareth meant by that.

**Comment by schra** [comment]
Thanks again for your PR!

**Pull request merged**